src/crossroads/main.py

```diff
@@ -56,7 +56,18 @@ def main_page():
     return render_template('index.html', form=form, crmap=google_map, )
 
 def locations_page(first_user_raw_loc, second_user_raw_loc, first_user_name, second_user_name):
-    coordinates = generate_coordinates(first_user_raw_loc, second_user_raw_loc)
+    final_lists = json_parser(first_user_raw_loc), json_parser(second_user_raw_loc)
```

Comment:
You are using `final_lists` in many places in this function however weach time it has different content. Name them accordingly